### DISCUSSION TITLE
Dunst notifications window opacity

### DISCUSSION THREAD

**Discussion by omnigenous** [hyprwm/Hyprland#7418] [Q&A] [answered]
Can't figure out how to add opacity to Dunst.

I have tried:

```
windowrulev2 = opacity 0.1 0.1,class:^(Dunst)$
```

But then I realized I could even found dunst in `hyprctl clients`.
Setting `transparency` in `dunstrc` only works for x11.

How can I do this in Hyprland?

**Comment by vaxerski** [comment]
dunst is a layer. You have to style it within its config

**Reply by omnigenous** [reply]
So this is not possible with Hyprland? Because dunst docs says this:
```
# The transparency of the window.  Range: [0; 100].
# This option will only work if a compositing window manager is
# present (e.g. xcompmgr, compiz, etc.). (X11 only)
transparency = 50
```

**Reply by vaxerski** [reply]
if dunst can't change its own opacity, then no, not possible.

**Comment by pbogut** [comment, marked as answer]
Its bit old, but in case you serach fo it as I did:

You can add alpha to the color:
`background = "#000000AA'`

https://github.com/dunst-project/dunst/blob/master/docs/dunst.5.pod#colors

**Reply by omnigenous** [reply]
Thank you!
